CONTRIBUTING.md

```diff
@@ -70,8 +70,8 @@ This project and everyone participating in it is governed by a [Code of Conduct]
 
 * Use tabs for indentation, spaces for alignment.
 * Lines should not exceed 120 columns.
-* To make sure the formatting is consistent, please use version 11.0.1, installable through `python3 -m pip install clang-format==11.0.1`
-* `clang_format` enforces these rules automatically, use `make format-fix` to run the formatter.
+* To make sure the formatting is consistent, please use version 10.0.1, installable through `python3 -m pip install clang-format==10.0.1.1`
```

Comment:
This seems to be the version that our CI uses

![image](https://github.com/duckdb/duckdb/assets/1405026/1d89818a-325c-41ff-8ebe-90fb2d96952b)